src/monitor.ts

```diff
@@ -7,17 +9,17 @@ import {
   EmbeddingEventDataFactory,
 } from './eventDataFactory';
 
+
 export interface MonitorOpenAIOptions extends EventClientOptions {
   applicationName: string;
 }
 
 export const monitorOpenAI = (
-  openAIApi: OpenAIApi,
+  openAIApi: any,
```

Comment:
We don't need this parameter anymore. The next version of the SDK should have a breaking change.

Reply:
we need in order it get the API key

Reply:
and all the configuration object

Reply:
Right, so let's add proper typing `OpenAIApi | langchain`